src/Messages.sol

```diff
@@ -38,7 +38,9 @@ library ConnectorMessages {
         /// 13 - Collect investment
         CollectInvest,
         /// 14 - Collect Redeem
-        CollectRedeem
+        CollectRedeem,
+        /// 15 - Schedule a spell to be relied on gateway (Admin only)
+        ScheduleRely
```

Comment:
I think we should call this `ScheduleAddAdmin` or something like that. `Rely` is Solidity specific naming, while the Connector message format should be more generic. Assume we build a Cosmos Connectors implementation at some point, then this would not use the rely naming.

Reply:
Is it okay to use `Rely` within gateway, since that will only exist in an EVM context?

Reply:
Yeah of course! This was just in relation to/from the message encoding to Centrifuge Chain